libs/clients/transport-authority/vehicle-codetables/src/lib/vehicleCodetablesClient.config.ts

```diff
@@ -0,0 +1,28 @@
+import { defineConfig } from '@island.is/nest/config'
+import * as z from 'zod'
+
+const schema = z.object({
+  xroadBaseUrl: z.string(),
+  xroadClientId: z.string(),
+  xroadPath: z.string(),
+})
+
+export const VehicleCodetablesClientConfig = defineConfig<
+  z.infer<typeof schema>
+>({
+  name: 'VehicleCodetablesClient',
+  schema,
+  load(env) {
+    return {
+      xroadBaseUrl: env.required('XROAD_BASE_PATH', 'http://localhost:8081'),
+      xroadClientId: env.required(
```

Comment:
Fyi. We have a shared config file for these two. Can check some of the other clients how we use it.